lib/models/lookup.js

```diff
@@ -58,7 +57,7 @@ function LookupModelFactory (
             },
             {
                 keys: { ipAddress: 1},
-                options: { unique: true }
+                options: { unique: false }
```

Comment:
@yyscamper , I would believe ipAddress should be unit in lookup table. When will it be NOT unique?